jdaviz/configs/imviz/tests/test_simple_aper_phot.py

```diff
@@ -357,8 +358,8 @@ def test_annulus_background(imviz_helper):
     assert_allclose(phot_plugin.background_value, 5.745596129482831)  # Changed
 
     # Draw ellipse on another object
-    # EllipsePixelRegion(center=PixCoord(x=20.5, y=37.5), width=41, height=15)
-    imviz_helper._apply_interactive_region('bqplot:ellipse', (0, 30), (41, 45))
+    # EllipsePixelRegion(center=PixCoord(x=20.5, y=20.5), width=41, height=15)

```

Comment:
Why did this one change drastically? When I wrote this, I designed it to be `EllipsePixelRegion(center=PixCoord(x=20.5, y=37.5), width=41, height=15)` but looks like you have to move the center point down by a lot, why?

Reply:
typo